WordPress/Classes/ViewRelated/Ratings/AppFeedbackPromptView.swift

```diff
@@ -35,45 +36,76 @@ class AppFeedbackPromptView: UIView {
         label.textAlignment = .center
         label.numberOfLines = 0
         label.adjustsFontForContentSizeCategory = true
         label.font = textFont
         label.text = NSLocalizedString("What do you think about WordPress?",
                                        comment: "This is the string we display when prompting the user to review the app")
-        container.addSubview(label)
+        addSubview(label)
 
-        leftButton.frame = CGRect(x: container.bounds.midX - 135.0, y: 50.0, width: 130.0, height: 30.0)
+        // Stack O'Buttons
+        buttonStack.translatesAutoresizingMaskIntoConstraints = false
+        buttonStack.axis = .horizontal
+        buttonStack.spacing = 10.0
+        addSubview(buttonStack)
+
+        // Yes Button
+        leftButton.translatesAutoresizingMaskIntoConstraints = false
         leftButton.backgroundColor = UIColor(red: 0.0, green: 170/255.0, blue: 220/255.0, alpha: 1.0)
-        leftButton.layer.cornerRadius = 4
-        leftButton.layer.masksToBounds = true
+        leftButton.tintColor = .white
         leftButton.setTitle(NSLocalizedString("I Like It",
                                               comment: "This is one of the buttons we display inside of the prompt to review the app"),
                             for: .normal)
         leftButton.setTitleColor(UIColor.white, for: .normal)
         leftButton.titleLabel?.font = textFont
         leftButton.addTarget(self, action: #selector(self.leftButtonTouched), for: .touchUpInside)
-        container.addSubview(leftButton)
+        buttonStack.addArrangedSubview(leftButton)
 
-        rightButton.frame = CGRect(x: container.bounds.midX + 5, y: 50.0, width: 130.0, height: 30.0)
+        // Could be Better Button
+        rightButton.translatesAutoresizingMaskIntoConstraints = false
         rightButton.backgroundColor = UIColor(red: 144/255.0, green: 174/255.0, blue: 194/255.0, alpha: 1.0)
-        rightButton.layer.cornerRadius = 4
-        rightButton.layer.masksToBounds = true
+        rightButton.tintColor = .white
         rightButton.setTitle(NSLocalizedString("Could Be Better",
                                                comment: "This is one of the buttons we display inside of the prompt to review the app"),
                              for: .normal)
         rightButton.setTitleColor(UIColor.white, for: .normal)
         rightButton.titleLabel?.font = textFont
         rightButton.addTarget(self, action: #selector(self.rightButtonTouched), for: .touchUpInside)
-        container.addSubview(rightButton)
+        buttonStack.addArrangedSubview(rightButton)
+
+        setNeedsUpdateConstraints()
+    }
+
+    override func updateConstraints() {
+        setupConstraints()
+        super.updateConstraints()
+    }
+
+    func setupConstraints() {
+        guard constraintsAdded == false else {
+            return
+        }
+        constraintsAdded = true
```

Comment:
I wasn't sure because the docs seemed to imply that adding constraints in a `UIView` subclass had to happen in `updateConstraints()` but it seems to work fine from `init()` ¯\_(ツ)_/¯